lib/Extension/Debug/Model/CommandDocumentor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Phpactor\Extension\Debug\Model;
+
+use Phpactor\MapResolver\Resolver;
+use ReflectionClass;
+use RuntimeException;
+use Phpactor\Extension\Rpc\Handler;
+
+class CommandDocumentor implements DocumentorInterface
```

Comment:
`RpcCommand` documentor to distinguish it from LSP commands

Reply:
Fixed